src/app/hearing-part/components/hearing-search-table/hearing-search-table.component.ts

```diff
@@ -68,4 +69,9 @@ export class HearingSearchTableComponent implements OnInit, OnChanges {
     buildViewHearingUrl(id: string) {
         return `/home/hearing/${id}`;
     }
+
+    canEdit(hearing: FilteredHearingViewmodel) {
```

Comment:
We probably shouldn't put this logic here, we could reuse this possibleActions on hearing which is calculated on backend. There's a need of adding new operations like if can be deleted and amended. Also there is a question whether we want to send these possibleActions together with entity or should it be like a separate endpoint which you would call after clicking on dropdown next to hearing to get the possibleActions then.